sycl/include/sycl/ext/oneapi/experimental/bfloat16_math.hpp

```diff
@@ -352,6 +406,18 @@ fma(T1 x, T2 y, T3 z) {
     return res;                                                                \
   }
 
+#if defined(__SYCL_DEVICE_ONLY__) && (defined(__SPIR__) || defined(__SPIRV__))
+#define BFLOAT16_MATH_FP32_WRAPPERS_VEC(op)                                    \
+  /* Overload for BF16 vec and swizzles. */                                    \
```

Comment:
https://www.llvm.org/docs/CodingStandards.html#comment-formatting:

> `//` is preferred for normal comments.

Reply:
AFAIK, we can not use `//` inside macro. https://stackoverflow.com/questions/24751387/can-i-comment-multi-line-macros

Reply:
Oh, totally right. Didn't realize it was within the macro. You can ignore this.